framework/db/Command.php

```diff
@@ -780,6 +780,16 @@ public function execute()
 
             $this->pdoStatement->execute();
             $n = $this->pdoStatement->rowCount();
+            
+            // https://bugs.php.net/bug.php?id=61613
+            // Make sure that supported drivers throw an exception in case a statement of a multi-query failed
+            try {
+                while ($this->pdoStatement->nextRowSet()) {}
+            } catch (\Exception $e) {
+                if (strpos($e->getMessage(), 'SQLSTATE[IM001]') === false) {
```

Comment:
@kidol This looks like a very specific error message. Any references for this message? And are you sure, this will work with different DBMS?